src/frontend/komponenter/Oppgavebenk/OppgavelisteSaksbehandler.tsx

```diff
@@ -17,10 +16,22 @@ const OppgavelisteSaksbehandler: React.FunctionComponent<IOppgavelisteSaksbehand
     oppgave,
     innloggetSaksbehandler,
 }) => {
-    const { fordelOppgave, tilbakestillFordelingPåOppgave } = useOppgaver();
-    const [feilmelding, setFeilmelding] = React.useState<string>();
-    const [erTilbakestilt, setErTilbakestilt] = React.useState<boolean>(false);
+    const { fordelOppgave, tilbakestillFordelingPåOppgave, oppdaterOppgave } = useOppgaver();
+    const [feilmelding, settFeilmelding] = React.useState<string>();
+    const [erTilbakestilt, settErTilbakestilt] = React.useState<boolean>(false);
     const { sjekkTilgang } = useApp();
+    const oppgaveRef = useRef<IOppgave | null>(null);
+
+    useEffect(() => {
+        if (oppgaveRef.current === null) {
+            oppgaveRef.current = oppgave;
+        }
+        if (oppgaveRef.current.id !== oppgave.id) {
+            settFeilmelding('');
+            settErTilbakestilt(false);
+        }
+        oppgaveRef.current = oppgave;
+    }, [oppgave]);
```

Comment:
Hva er behovet her? Virker litt magisk.

Reply:
Tok en stund før jeg forstod logikken i oppgavebenken selv. Men den lager én komponent for hver linje, og når man navigerer byttes oppgaven ut. Så linje en vil da f.eks. få oppgave 1 og 16.. så her tar jeg var på hva den var sist i en ref, og så nullstiller vi erTilbakestilt når den får en oppgave som er ulik den forrige.

Reply:
Aha, magic 🤯  Nice 🙏